doc/controllers_index.rst

```diff
@@ -72,3 +72,12 @@ In the sense of ros2_control, broadcasters are still controllers using the same
    IMU Sensor Broadcaster <../imu_sensor_broadcaster/doc/userdoc.rst>
    Joint State Broadcaster <../joint_state_broadcaster/doc/userdoc.rst>
    Range Sensor Broadcaster <../range_sensor_broadcaster/doc/userdoc.rst>
+
+
+Common Controller Parameters
+****************************
+
+Every controller and broadcaster have a few common parameters that are necessary upon initialization. These parameters are `update_rate` and `is_async`.
```

Comment:
```suggestion
Every controller and broadcaster has a few common parameters that are optional upon initialization:
```

Reply:
These parameters are needed to be set before configuring the controller.

Reply:
I updated my suggestion.

I saw now that the docs lack of proper description of the lifecycle changes/transitions (nothing to link to).

Reply:
Hello, I reformatted a bit. Can you verify?

Every controller and broadcaster has a few common parameters. They are optional, but if needed they have to be set before ``onConfigure`` transition to ``inactive`` state, see `lifecycle documents <https://design.ros2.org/articles/node_lifecycle.html>`__. Once the controllers are already loaded, this transition is done using the ``configure_controller`` service of the controller_manager: